src/exo/__init__.py

```diff
@@ -1,9 +1,12 @@
 from .API import Procedure, compile_procs, proc, instr, config
 from .LoopIR_scheduling import SchedulingError
+from .parse_fragment import ParseFragmentError
 from .configs import Config
 from .memory import Memory, DRAM
 from . import query_asts as QAST
 
+from . import stdlib
```

Comment:
I am really not sure if it's a good idea to name our standard library `stdlib`. Maybe just `primitives` or `ops` ? That's more consistent with e.g., `apps/` and `platforms/`, no?

Reply:
The idea is that `stdlib` lives under `exo` (i.e. `exo.stdlib`) in the Python module namespace.  Ultimately, I expect `stdlib` to hold things like "standard memories" excluding `apps` and `platforms` which are not part of the standard `exo-lang` distribution on PyPI.  We can create a second PyPI distribution for things like `platforms` libraries.

Reply:
Hmmm I think `stdlib` is not saying anything, shouldn't users be able to import DRAM etc. and all the scheduling primitives directly with `from exo import *` ? I just think `exo.stdlib` should be just as simple as `exo`. And we can have peripheral code such as `builtins` and `platforms` separately and be accessed with `exo.builtins`

Reply:
So, there will be a lot of scheduling primitives, not all of which will be atomic (e.g. `repeat`) and I expect that list to grow somewhat.  Generally I would expect `from exo import *` to get you things like `proc` that you'll need to use regularly, and then you may want to specifically bind the namespace of scheduling operations to a shorthand like `S` or `XS` (exo scheduling?) via
```
from exo.stdlib import scheduling as S
```

Reply:
`builtins` may be worth having under `exo` directly instead of `stdlib`.  I'm not sure.  As far as `platforms`, that shouldn't even be under `exo` in the namespace at all, and ultimately shouldn't be packaged with `exo-lang` in PyPI.  We can create a separate `exo-lang-platforms` package if we want to have standard "platform support" with lower guarantees of quality than the core `exo-lang` distribution.

Reply:
Shouldn't it be `exo.scheduling` then? I'm not sure why having `stdlib` in the middle is useful

Reply:
My reply to this got scrambled.  My ultimate plan for this is to move most of the current contents of the `exo` directory into a subdirectory called something like `compiler`.  At that point it may make sense to move `scheduling` to `exo.scheduling` and likewise for other submodules that we want users to import.  However, right now, there would be no distinction between `exo.scheduling` and `exo.LoopIR` which definitely should not be imported by user code.